sdgym/synthesizers/base.py

```diff
@@ -54,8 +54,11 @@ def get_trained_synthesizer(self, data, metadata):
             obj:
                 The synthesizer object.
         """
-        metadata_class = MultiTableMetadata() if 'tables' in metadata else SingleTableMetadata()
-        metadata = metadata_class.load_from_dict(metadata)
+        metadata_object = Metadata()
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore', UserWarning)
```

Comment:
Suppressing [this](https://github.com/sdv-dev/SDV/blob/363d8bdca29fa26c3626d86bb535a7ca656a8671/sdv/metadata/metadata.py#L131-L132) warning.